### ISSUE TITLE
Warning: Cannot read property 'name' of null - caused by valid-jsdoc rule

### ISSUE THREAD

**Issue by Merott** [buzinas/tslint-eslint-rules#238]
- `tslint 5.4.3`
- `tslint-eslint-rules 4.1.1`

TSlint config:

```json
"valid-jsdoc": [true, {
  "requireReturn": false,
  "requireReturnType": false
}],
```

Code:

```ts
class MyPage {
  /**
   * Navigate to the page
   * @returns a promise for the browser's navigation
   */
  public async navigateTo() { }
}
```

Getting this warning:

```
Warning: Cannot read property 'name' of null
```

**Comment by sanex3339** [comment]
When i can expect release with this fix?
